client/containers/Builder.js

```diff
@@ -150,6 +150,22 @@ class Builder extends Component {
           showSaveCompleteModal: true
         })
       })
+      .catch(err => console.log(err))
+  }
+
+  deleteBuild = id => {
+    fetch('/api/userConfigs/' + id, {
+      method: 'DELETE'
+    })
+      .then(() => {
+        const copy = [...this.state.buildHistory]
+        copy.find((elem, index, array) => {
```

Comment:
Don't use `find` for changing the array. Find is for returning the item you want.

```js
const buildIndex = copy.findIndex(build => build.id === id)
copy.splice(buildIndex, 1)
```